Description: Install scripts for creating artwork.
 There are two scripts for creating artwork files included in the wmcdplay
 source.  Previously, they were not included in the Debian package.
 .
 Beginning in version 1.0beta1-13, they are installed in /usr/lib/wmcdplay.
 This patch updates the makeall script to reflect this location and updates
 the ARTWORK documentation file accordingly.
Author: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: 2014-08-01

--- a/XPM/makeall
+++ b/XPM/makeall
@@ -1,3 +1,3 @@
 #!/bin/sh
 echo "Making all artwork files"
-find ./* -type d -exec ./makeart {} \;
+find ./* -type d -exec /usr/lib/wmcdplay/makeart {} \;
--- a/ARTWORK
+++ b/ARTWORK
@@ -247,10 +247,29 @@
 ARTWORK FILES
 =============
 
-All the above files should be placed in an appropriately named directory in wmcdplay/XPM/
-and the script wmcdplay/XPM/makeall should be executed.
-You should end up with a new .art file in wmcdplay/XPM/
-You should be able to load wmcdplay with this artwork using the "-f artwork_file" command
-line argument
-You should also be able to make your new artwork the compile-time default by changing the
-symbolic link wmcdplay/XPM/standard.art point to it, and recompiling
\ No newline at end of file
+   Suppose you have created the files as outlined above in the directory
+   ~/wmcdplay/foo.  Then you can run
+
+      cd ~/wmcdplay
+      /usr/lib/wmcdplay/makeart foo
+
+   You can then use your new artwork file by running
+
+      wmcdplay -f ~/wmcdplay/foo.art
+
+   Additionally, you can run
+
+      sudo mv ~/wmcdplay/foo.art /usr/share/wmcdplay
+
+   And then you only need to run
+
+      wmcdplay -f foo.art
+
+   If you have created multiple artwork files in separate subdirectories of
+   ~/wmcdplay, you can convert them to .art files all at once using
+
+      cd ~/wmcdplay
+      /usr/lib/wmcdplay/makeall
+
+   Finally, if you compile wmcdplay from source, you can replace the default
+   artwork file with another by pointing the symlink XPM/standard.art at it.
